source/scenario/mission3task1.vwf.yaml

```diff
@@ -36,8 +36,6 @@ properties:
     - [ "assets/video/Success3_800x600.mp4" ]
 
   #blockly
-  - setBlocklyContext:
-    - rover
```

Comment:
Why are we removing this?  It's probably not necessary unless we jump back from m4, so not critical - but is it hurting anything?